WordPress/Classes/Extensions/CLPlacemark+Formatting.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+import CoreLocation
+
+extension CLPlacemark
+{
+    /**
+     Returns a single string with the address information of a placemark formatted
+     */
```

Comment:
Nitpicky, but maybe use the three slash comment style for Swift files as recommended by @diegoreymendez